# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpful if you want to save parameters for different use-cases.
# In any way: Default values should go into the .if file!
#
# To generate a yaml file, use "rosrun rosinterface_handler generate_yaml /path/to/PathPublisher.if"
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

map_name: fitted_road_map.osm
center_x: 3.0
center_y: 5.0
radius_noise: 0.3
rotation_noise: 20
shift_noise: 0.2
path_length: 6          #whole path has to be larger than this length
drive_distance: 2     #recommand to be half of path_length
timer_rate: 20
verbosity: debug
point_distance: 0.02    #same to path generator


# following are env sensitive parameters
mode: train
env: carla
frame_id_map: map
frame_id_vehicle: base_link
least_points: 300
local_scope: 20       #local scope in real world (m) 
kox_shift: -0.2
